keras_hub/src/utils/keras_utils.py

```diff
@@ -104,3 +104,38 @@ def running_on_gpu():
         import torch
 
         return torch.cuda.is_available()
+
+
+def gpu_supports_fused_attention_op():
+    deny_list = ["t4"]
+    available_gpu = [gpu.upper() for gpu in get_gpu_name()]
+    for denied_gpu in deny_list:
+        if any(denied_gpu.upper() in gpu for gpu in available_gpu):
```

Comment:
Why do you need a `T4 in string` comparison instead of equality? Is there a suffix of on some backends?

Reply:
the output of the device name is something long like this `'NVIDIA A100-SXM4-40GB'`. I am trying to check if name T4( or anything on deny list) is in the GPU name